inc/devicesensor.class.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2017 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access directly to this file");
+}
+
+/*
+ * @since version 9.2
+ */
+class DeviceSensor extends CommonDevice {
+
+   static protected $forward_entity_to = array('Item_DeviceSensor', 'Infocom');
+
+   static function getTypeName($nb=0) {
+      return _n('Sensors', 'Sensors', $nb);
+   }
+
+
+   function getAdditionalFields() {
+
+      return array_merge(parent::getAdditionalFields(),
+                         array(array('name'  => 'devicesensortypes_id',
+                                     'label' => __('Type'),
+                                     'type'  => 'dropdownValue')));
+   }
+
```

Comment:
do not use array but [] instead (https://patch-diff.githubusercontent.com/raw/glpi-project/docdev/pull/53.diff)